ui/raidboss/data/07-dt/trial/zoraal-ja.txt

```diff
@@ -0,0 +1,188 @@
+### Everkeep
+# ZoneId: 1200
+
+# -ii 9351 93A6 9364 9365 9366 9367 937A 936A 936B 936C 936E 936F 9375 9376 8AEF
+# -ic "Fang"
+# -p 9370:300
+
+hideall "--Reset--"
+hideall "--sync--"
+
+##### PHASE 1 #####
+
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+8.8 "--sync--" StartsUsing { id: "934B", source: "Zoraal Ja" }
+13.8 "Soul Overflow" Ability { id: "934B", source: "Zoraal Ja" }
+21.1 "--center--" Ability { id: "98E1", source: "Zoraal Ja" }
+27.4 "Double-edged Swords (first)" Ability { id: "9352", source: "Zoraal Ja" } window 5,1
+29.7 "Double-edged Swords (second)" #Ability { id: "9352", source: "Zoraal Ja" }
+37.9 "Patricidal Pique" Ability { id: "9353", source: "Zoraal Ja" }
+41.1 "--sync--" Ability { id: "98E1", source: "Zoraal Ja" }
```

Comment:
Here and elsewhere, not all 98E1 abilities got marked as `--center--`

Reply:
Thanks for catching!  Will update this before I merge. 👍